tests/test_rds.py

```diff
@@ -844,6 +844,47 @@ def test_rds_snapshot_unmark(self):
         tag_map = {t['Key']: t['Value'] for t in tags['TagList']}
         self.assertFalse('maid_status' in tag_map)
 
+    def test_rds_public_accessible_enable(self):
```

Comment:
can you drop this test, its a straight pass through on the config value, its not really adding to the testing imo, and rds tests carry alot of recorded json weight/baggage.